staging/src/k8s.io/apiserver/pkg/endpoints/handlers/create.go

```diff
@@ -144,7 +145,12 @@ func createHandler(r rest.NamedCreater, scope *RequestScope, admit admission.Int
 				return
 			}
 
-			obj, err = scope.FieldManager.Update(liveObj, obj, managerOrUserAgent(options.FieldManager, req.UserAgent()))
+			fieldManager, err := defaultFieldManager(obj, options.FieldManager, req.UserAgent())
```

Comment:
Potentially, you could leave the userAgent out of the defaultFieldManager and still call `managerOrUseAgent` when calling `Update`?

Reply:
Then you can reuse defaultFieldManager for Apply, and do something different if it's not specified.